rasa/core/policies/ted_policy.py

```diff
@@ -323,9 +327,13 @@ def __init__(
         """Declare instance variables with default values."""
         self.split_entities_config = rasa.utils.train_utils.init_split_entities(
             kwargs.get(SPLIT_ENTITIES_BY_COMMA, SPLIT_ENTITIES_BY_COMMA_DEFAULT_VALUE),
-            self.defaults[SPLIT_ENTITIES_BY_COMMA],
+            self.defaults.get(
+                SPLIT_ENTITIES_BY_COMMA, SPLIT_ENTITIES_BY_COMMA_DEFAULT_VALUE
+            ),
```

Comment:
Isn't it always `SPLIT_ENTITIES_BY_COMMA_DEFAULT_VALUE`?

Reply:
It's an invalid parameter for `IntentTEDPolicy` and hence it wouldn't exist in the defaults of `IntentTEDPolicy` which is why this line will error out when instantiating `IntentTEDPolicy`.